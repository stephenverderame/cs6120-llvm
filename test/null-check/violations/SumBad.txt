[31mSafety Violation[00m: Use of potentially null pointer in load at SumBad.cc:11
Because an index could not be proven to be in range, the following counterexample was found:
	[1msize[0m = 64
