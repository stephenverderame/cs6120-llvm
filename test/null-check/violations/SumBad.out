[31mSafety Violation[00m: Use of potentially null pointer in load at SumBad.cc:11
